misk/src/main/kotlin/misk/environment/EnvironmentModule.kt

```diff
@@ -1,10 +1,26 @@
 package misk.environment
 
 import com.google.inject.AbstractModule
+import misk.logging.getLogger
 
-class EnvironmentModule(private val environment: Environment) : AbstractModule() {
+private val logger = getLogger<EnvironmentModule>()
+
+class EnvironmentModule(val environment: Environment) : AbstractModule() {
     override fun configure() {
         bind(Environment::class.java).toInstance(environment)
     }
 
+    companion object {
+        @JvmStatic
+        fun fromEnvironmentVariable(): EnvironmentModule {
+            val environmentName = System.getenv("ENV")
+            val environment = if (environmentName != null) {
+                Environment.valueOf(environmentName)
```

Comment:
We'd want to ensure `environmentName` is upper-cased since `Enum#valueOf()` isn't case-insensitive.

Reply:
Ah gotcha, so rather than forcing the variable value to be uppercased, we do it for them? Sounds good, will update.